_data/documentation.yml

```diff
@@ -21,7 +21,7 @@ PySAL model:
   mgwr:  https://mgwr.readthedocs.io/en/latest/
   spglm: https://github.com/pysal/spglm
   spint: https://github.com/pysal/spint
-  spreg: https://github.com/pysal/spreg
+  spreg: https://spreg.readthedocs.io/
```

Comment:
Should this also include `en/latest/`?

Reply:
it will jump to the latest automatically.